posthog/models/organization.py

```diff
@@ -222,9 +222,14 @@ def update_available_product_features(self) -> list[ProductFeature]:
 
         return self.available_product_features
 
+    def get_available_feature(self, feature: Union[AvailableFeature, str]) -> dict:
```

Comment:
i moved get_available_feature here, seems like better fit

Reply:
Nice, this is good 👍

Reply:
Need to update return to be `dict | None` I think